src/pages/embedded/index.ts

```
@@ -448,55 +438,44 @@ window.addEventListener("DOMContentLoaded", function () {
-							const maximizePlayerButton = document.querySelector<HTMLButtonElement>("video.html5-main-video");
-							if (!maximizePlayerButton) return;
-							// Get the video element
-							const videoElement = document.querySelector<HTMLVideoElement>("video.html5-main-video");
-							// If video element is not available, return
-							if (!videoElement) return;
-							const videoContainer = document.querySelector<YouTubePlayerDiv>("video.html5-main-video");
-							if (!videoContainer) return;
-							if (videoContainer.classList.contains("maximized_video_container") && videoElement.classList.contains("maximized_video")) {
-								maximizePlayer();
-							}
+						if (maximizePlayerButtonEnabled) return await addMaximizePlayerButton();
```

ESLint

[error] Insert `␍`

(prettier/prettier)